AssetTracker/apps/companyusers/migrations/0001_initial.py

```diff
@@ -18,8 +18,8 @@ class Migration(migrations.Migration):
         migrations.CreateModel(
             name='CompanyUser',
             fields=[
-                ('User', models.OneToOneField(on_delete=django.db.models.deletion.CASCADE, primary_key=True, serialize=False, to=settings.AUTH_USER_MODEL)),
-                ('Company', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='companies.Company')),
+                ('user', models.OneToOneField(on_delete=django.db.models.deletion.CASCADE, primary_key=True, serialize=False, to=settings.AUTH_USER_MODEL)),
```

Comment:
To add space before and after each '=' sign wherever required

Reply:
This one is automatically generated file when you make migrations